cmd/konflux-apply/README.md

```diff
@@ -0,0 +1,56 @@
+# Konflux Apply
+
+The Konflux manifests are applied automatically via the [konflux-apply](../../.github/workflows/apply-konflux-manifests.yaml) Github Workflow.
+
+For this workflow, we use a token for Konflux setup as described in [the docs](https://gitlab.cee.redhat.com/konflux/docs/users/-/blob/main/topics/getting-started/getting-access.md#logging-to-the-internal-cluster-with-a-token), with the following service account, role & rolebindings (those manifest can be found in [./manifests](./manifests) too):
```

Comment:
> those manifest can be found in [./manifests](./manifests) too

I would use them for documentation instead of duplicating ?

Reply:
Would be great, if we could have directly referenced the files (and shown the content). But changed it to only a `kubectl apply ...`